### PR TITLE
Add amp-ad support for PolymorphicAds

### PR DESCRIPTION
**Pull request by fullspeed-sp**
"PolymorphicAds" added to amp-ad.
Sample tag displays image banner if it works.

Please review and merge changes.

### CODE REVIEW

File: examples/ads.amp.html [hunk 1]
```diff
@@ -970,6 +971,16 @@ <h2 id="plista">Plista responsive widget</h2>
     <div placeholder></div>
     <div fallback></div>
   </amp-embed>
+  
+  <h2 id="polymorphicads">polymorphicAds</h2>
+  <amp-ad width=320 height=50
+      type="polymorphicads"
+      data-adunit="7c0b3ae742beccf94f7726ea832277a2"
+      data-params='{"testMode": "true"}'
+      >
```
Comment (lannka) [inline examples/ads.amp.html]: move to previous line

Comment (fullspeed-sp) [inline examples/ads.amp.html]: Thank you for review.

Closing bracket moved to previous line in new commit.
Also added renderStartImplemented in _config.js.

**Comment by fullspeed-sp** [comment]
@lannka 
Hi, pull request fixed. Please review again.

**Comment by lannka** [comment]
@fullspeed-sp sorry for the late reply. did you make it work locally? I got a 500 error.

screen shot 2017-05-04 at 3 14 38 pm

**Comment by fullspeed-sp** [comment]
@lannka

Thank you for the review.
Some modification made to parameter names.
Please review again, as quickly as possible.

**Comment by lannka** [comment]
@fullspeed-sp would you follow the [guide](https://github.com/ampproject/amphtml/blob/master/ads/README.md#developer-guidelines-for-a-pull-request) and test it locally?

I got other errors complaining wrong attributes. You will use dash separated form: `data-ad-unit-id`.
screen shot 2017-05-09 at 3 21 12 pm

**Comment by fullspeed-sp** [comment]
@lannka

Parameter name edited back and made modification in our system.
I'm really sorry for lack of confirmation.

Please check it's working again.

File: examples/ads.amp.html [hunk 2]
```diff
@@ -920,6 +921,16 @@ <h2 id="plista">Plista responsive widget</h2>
...
   </amp-embed>
 
+  <h2 id="polymorphicads">polymorphicAds</h2>
+  <amp-ad width=320 height=50
+      type="polymorphicads"
+      data-adunit="7c0b3ae742beccf94f7726ea832277a2"
+      data-params='{"testMode": "true"}'
+      >
+    <div placeholder></div>
+    <div fallback></div>
```
Comment (lannka) [inline examples/ads.amp.html]: please remove `placeholder` and `fallback`. also move the `>` to the previous line.

**Comment by fullspeed-sp** [comment]
@lannka

Oops.
Made some tweaks.
Please check again.

**Comment by lannka** [comment]
@fullspeed-sp please fix the linter error

**Comment by fullspeed-sp** [comment]
@lannka

Fixed according to linter log.
I think it's all clear. 

Sorry to take your time. Please review again.

**Pull request merged**